Sources/Controllers/TargetMenu/Favorites/OAFavoriteListViewController.mm

```diff
@@ -1003,18 +1004,22 @@ - (OAPointTableViewCell *) setupPoiIconForCell:(OAPointTableViewCell *)cell with
 - (UITableViewCell*)getActionCellForRowAtIndexPath:(NSIndexPath *)indexPath
 {
     NSDictionary *item = _data[indexPath.section][indexPath.row];
-    OAIconTextTableViewCell* cell;
-    cell = (OAIconTextTableViewCell *)[self.favoriteTableView dequeueReusableCellWithIdentifier:[OAIconTextTableViewCell getCellIdentifier]];
+    OARightIconTableViewCell* cell;
+    cell = (OARightIconTableViewCell *)[self.favoriteTableView dequeueReusableCellWithIdentifier:[OARightIconTableViewCell getCellIdentifier]];
     if (cell == nil)
     {
-        NSArray *nib = [[NSBundle mainBundle] loadNibNamed:[OAIconTextTableViewCell getCellIdentifier] owner:self options:nil];
-        cell = (OAIconTextTableViewCell *)[nib objectAtIndex:0];
+        NSArray *nib = [[NSBundle mainBundle] loadNibNamed:[OARightIconTableViewCell getCellIdentifier] owner:self options:nil];
+        cell = (OARightIconTableViewCell *)[nib objectAtIndex:0];
     }
 
-    if (cell)
-    {
-        [cell.textView setText:[item objectForKey:@"text"]];
-        [cell.iconView setImage: [UIImage imageNamed:[item objectForKey:@"icon"]]];
+    if (cell) {
```

Comment:
Use C style curly brace (starting from the newline) here and in other places in this request.